packages/scxa-tsne-plot/html/Demo.js

```diff
@@ -187,7 +188,7 @@ class Demo extends React.Component {
     return(
       <div className={`row column expanded`}>
         <TsnePlotView
-          atlasUrl={`https://wwwdev.ebi.ac.uk/gxa/sc/`}
+          atlasUrl={`http://localhost:8080/gxa/sc/`}
```

Comment:
Please change this to `wwwdev` so that other developers don't need to run their local setup to test this component.